src/Chapter1.hs

```diff
@@ -428,7 +428,7 @@ task is to specify the type of this function.
 >>> squareSum 3 4
 49
 -}
-
+squareSum :: Int->Int->Int
```

Comment:
Correct! 💯 
Although, in Haskell all type operators and generally operators are separated with spaces like this:

```suggestion
squareSum :: Int -> Int -> Int
```